### DISCUSSION TITLE
Επιπλέον ημέρες και ώρες παρουσίασης

### DISCUSSION THREAD

**Discussion by p17simt** [courses-ionio/hci#1709] [Announcements]
Καλησπέρα, επειδή η μόνη χρονοθυρίδα παρουσίασης δεν με βολεύει καθώς θα δουλεύω εκείνη την ώρα, υπάρχει περίπτωση να ανοίξουν νέες χρονοθυρίδες τελικής παρουσίασης;

**Comment by epidrome** [comment]
την προηγούμενη εβδομάδα είχε 10άδες διαθέσιμες θέσεις που έμειναν κενές, ειδικά Δευτέρα-Τρίτη-Τετάρτη

φυσικά θα έχει διαθέσιμες θέσεις και την επόμενη εβδομάδα αλλά με μειωμένο βαθμό όπως ισχύει για όλα τα εκπρόθεσμα παραδοτέα
